### DISCUSSION TITLE
Πιθανό template για Jekyll

### DISCUSSION THREAD

**Discussion by atsiano81** [sdy60-2022/.github#30] [Ideas]
Παραθέτω πιθανα template jekyll για βιογραφικό
https://jekyll-themes.com/some/
https://jekyllthemes.io/theme/resume-template
από τα πιο απλά

**Comment by Katethoma** [comment]
Καλησπέρα. Μπορούμε να επιλέξουμε και κάποιο άλλο ή θα είναι παραφωνία ?

**Reply by atsiano81** [reply]
νομίζω ότι μας είπε ότι ο καθένας μπορεί να έχει το δικό
